### PR TITLE
Don't clean up files on image-based tests since the instance will be …

### PR DESCRIPTION
**Pull request by pwittrock**
…deleted anyway and this is flaky on CoreOS.

### CODE REVIEW

**Comment by pwittrock** [comment]
For #24297

**Comment by pwittrock** [comment]
@k8s-bot test this issue: #23361

**Comment by pwittrock** [comment]
@k8s-bot unit test this issue: #24355

File: test/e2e_node/e2e_node_suite_test.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ type LogReporter struct{}
 func (lr *LogReporter) SpecSuiteWillBegin(config config.GinkgoConfigType, summary *types.SuiteSummary) {
 	b := &bytes.Buffer{}
 	b.WriteString("******************************************************\n")
-	glog.V(2).Infof(b.String())
+	glog.V(0).Infof(b.String())
```
Comment (mikedanese) [inline test/e2e_node/e2e_node_suite_test.go]: nit: level 0 is pretty much equivalent to just glog.Infof

**Comment by mikedanese** [comment]
LGTM

File: test/e2e_node/e2e_node_suite_test.go (hunk 1 above)
Comment (pwittrock) [inline test/e2e_node/e2e_node_suite_test.go]: Thanks.  Should I fix this here, or just do as a follow up when working on the next piece?

**Pull request merged**